opencr.cert

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this the server certificate? If yes, are you really publishing certificates in a git repo? This is like putting a password in the repo, I think this exposes a security whole

Reply:
Actually this is not supposed to be here. This was just a mistake.